homepage/Portfolio/index.js

```diff
@@ -1,50 +1,49 @@
-
 menu1_is_open = false;
 menu2_is_open = false;
 menu3_is_open = false;
 
 function open_on_click(number) {
-    num = number;
-    if (menu1_is_open) {
-        if (number == 1) {
-            document.getElementById("projekt_one_arrow").src = "img/Arrow_Right.png";
-            document.getElementById("projekt_one_container").style.display = "none";
-            menu1_is_open = false;
-        }
-    } else {
-        if (number == 1) {
-            document.getElementById("projekt_one_arrow").src = "img/Arrow_Down.png";
-            document.getElementById("projekt_one_container").style.display = "block";
-            menu1_is_open = true;
-        }
+  num = number;
+  if (menu1_is_open) {
+    if (number == 1) {
+      document.getElementById("projekt_one_arrow").src = "img/Arrow_Right.png";
```

Comment:
```suggestion
  const var = document.getElementById("projekt_one_arrow");
  if (menu1_is_open) {
    if (number == 1) {
      var.src = "img/Arrow_Right.png";
```

You can extract the elements into variables.